src/math/FGFunction.h

```diff
@@ -798,6 +798,8 @@ class FGFunction : public FGParameter, public FGJSBBase
   double cachedValue;
   std::vector <FGParameter_ptr> Parameters;
 
+  bool constForConstParams;
+
```

Comment:
Remove the `constForCostParams` member, it is not needed (see comment below in `FGFunction::IsConstant`)